contrib/flake.nix

```diff
@@ -3,93 +3,100 @@
 
```

Comment:
we can expand on the definition, something like "provides neovim nightly version for both users and developers"